catalog/imaging/imaging-postprocess-plugin-nitf/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
❓ Why is this needed?

Reply:
Not sure, I think it was auto-generated. It looks like its in a couple other places as well